app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFragment.kt

```diff
@@ -50,10 +50,11 @@ class PocketVideoFragment : Fragment() {
         val viewModel = ViewModelProviders.of(this, factory).get(PocketViewModel::class.java)
 
         viewModel.state.observe(viewLifecycleOwner, Observer<PocketViewModel.State> { state ->
-            state ?: return@Observer
             when (state) {
                 is PocketViewModel.State.Error -> { /* TODO: #769: display error screen */ }
                 is PocketViewModel.State.Feed -> adapter.setVideos(state.feed)
+                is PocketViewModel.State.NotDisplayed -> { }
```

Comment:
What does this do?

Reply:
If our state is `notDisplayed` then we shouldn't even be able to reach this fragment.  I can add a comment in explaining that.  Do you think we should throw an exception there as well?